modules/react/skeleton/lib/parts/skeletonText.tsx

```diff
@@ -20,55 +22,45 @@ export interface SkeletonTextProps extends React.HTMLAttributes<HTMLDivElement>
   backgroundColor?: string;
 }
 
-const Line = styled('div')<{
-  width: number | string;
-  height: number | string;
-  borderRadius: number;
-  backgroundColor: string;
-}>(({width, height, borderRadius, backgroundColor}) => {
+const Line = styled('div')<
+  {
+    width: number | string;
+    backgroundColor: string;
+  } & StyledType
+>(({backgroundColor, width}) => {
   return {
-    width: width,
-    height: height,
-    borderRadius: borderRadius,
-    backgroundColor: backgroundColor,
+    backgroundColor,
+    width,
+    borderRadius: TEXT_BORDER_RADIUS,
```

Comment:
In this case I felt it warranted to move non-default, static values into the styled component.